### ISSUE TITLE
Is there any way to provide parameters to included dsl files?

### ISSUE THREAD

**Issue by andrask** [ghale/gradle-jenkins-plugin#59]
In my gradle build I use the 

    	dsl fileTree('scripts').include('*.dsl')

method to include the dsls of jobs. Now the issue I'm facing is that I cannot reach the properties defined on the project like I would if I inlined the job.

What is the method of passing an environment to the dsl files?

**Comment by itavero** [comment]
I would like to know this too. Specifically passing entire objects would interest me (instead of just regular "string" parameters).

@Andrask did you manage to achieve this?

**Comment by andrask** [comment]
No, I tried many ways, even checked the corresponding code, but there doesn't seem to be any sane way to do it.
